README.md

```diff
@@ -33,6 +36,27 @@ Svelte v3 + v2
 </script>
 ```
 
+## Adaption for Sapper
```

Comment:
I wrote this module for Sapper. I use it in Sapper without this adaptation (and I wrote that section of the Sapper docs!) - are you sure it's necessary?

If it is perhaps I can help you debug. It shouldn't be.

Reply:
I'll re-do the install with devDependencies, and if its still a problem, I'll make a video of the problem.  In its current state, its deffo not working for me. 

I also did see one other change I forgot to add to this pull request, the alt text of the facebook icon is "google", will also do that tomorrow.

Reply:
@antony as described above. the --save-dev made all the difference. I'm gonna adapt the pull request to remove this, adapt the install instructions and change "google" to "facebook" in the <FacebookAuth> component.